source/dotnet/wholesale-api/CalculationResults/CalculationResults.Infrastructure/CalculationResultClient/DatabricksSqlResponse.cs

```diff
@@ -14,4 +14,4 @@
 
 namespace Energinet.DataHub.Wholesale.CalculationResults.Infrastructure.CalculationResultClient;
 
-public sealed record DatabricksSqlResponse(string State, IEnumerable<string[]> Rows);
+public sealed record DatabricksSqlResponse(string State, TableRows TableRows);
```

Comment:
I prefer the shorter name
```suggestion
public sealed record DatabricksSqlResponse(string State, TableRows Rows);
```